flux_led/__init__.py

```diff
@@ -622,9 +635,9 @@ def setRgb(self, r,g,b, persist=True):
             msg = bytearray([0x31])
         else:
             msg = bytearray([0x41])
-        msg.append(r)
-        msg.append(g)
-        msg.append(b)
+        msg.append(int(r))
```

Comment:
I see that you created a new method setRgbw(). Why not implement a new one getRgbw(). 

Did you test if this does not break the current getRgb() on nonwhite lights?